packages/core/src/stats/index.ts

```diff
@@ -87,3 +92,13 @@ export default class Stats {
     })
   }
 }
+
+export class NullStats implements Stats {
```

Comment:
Node doesn't currently use stats / metrics, but we might want to capture it in the future. Using this as the default instance when adding a context lets us avoid using optional chaining, which decreases compilation size overall.